sycl/include/CL/sycl/backend/level_zero.hpp

```diff
@@ -57,7 +57,7 @@ struct interop<backend::level_zero,
 };
 
 namespace detail {
-template <> class BackendReturn<backend::level_zero, kernel> {
+template <> struct BackendReturn<backend::level_zero, kernel> {
```

Comment:
Is it alignment with template declaration?

Reply:
Yes